test/unit/alignment/configuration/align_config_aligned_ends_test.cpp

```diff
@@ -48,7 +48,7 @@ using static_end_gap_types = ::testing::Types<front_end_first<std::true_type>,
                                               back_end_second<std::true_type>,
                                               back_end_second<std::false_type>>;
 
-TYPED_TEST_CASE(static_end_gap_test, static_end_gap_types);
+TYPED_TEST_SUITE(static_end_gap_test, static_end_gap_types, );
 
```

Comment:
> t's a bug that only affects `INSTANTIATE_TYPED_TEST_SUITE_P` [google/googletest#2271](https://github.com/google/googletest/issues/2271)

But this is not INSTANTIATE_TYPED_TEST_SUITE_P.

Reply:
Affects any variadic stuff (see bug report)